styles/styles.css

```diff
@@ -255,3 +255,49 @@ main .section.highlight {
   margin: 0;
   padding: 40px 0;
 }
+/* Day 1 (Part 2) Assignment #3 start */
+
+.light-grey{
+  background-color: #ccc;
+  height: 400px;
+  color: #000;
+  font-size: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center; 
+}
+
+/* Day 1 (Part 2) Assignment #3 end */
+
+
+/* Day 2 – Part 1  Assignment #1 start  */
+
+.imagesection img {
+  width: 100%; 
+  height: auto; 
+  display: block; 
+  max-width: 100%;
+}
+
+.button-container{
+  text-align: center;
+}
+
+.section.linkstyle .button-container .button {
+  background-image: linear-gradient(rgba(179, 132, 201, .84), rgba(152, 81, 244, 0.84) 50%);
+  color:#fff;             
+  padding: 10px 20px;        
+  text-decoration: none;     
+  border-radius: 25px;        
+  font-size: 16px;          
+  display: inline-block;     
+}
+
+.section.linkstyle .button-container .button:hover {
+  background-image: linear-gradient(rgba(98, 189, 219, 0.84), rgba(65, 156, 199, 0.84) 50%);
+  color:#000;
+}
+
+/* Day 2 – Part 1  Assignment #1 end  */
```

Comment:
Keep the clean CSS for generic styles, make sure it is reusable.

Reply:
CSS changed.